### PR TITLE
docs(swarm local address): add documentation spec for swarm.localAddrs

### LINKED ISSUE #296: Missing spec for swarm.localAddrs
https://ipfs.io/docs/commands/#ipfs-swarm-addrs-local

### PR DESCRIPTION
**Pull request by Prabhakar-Poudel**
closes #296

### CODE REVIEW

File: SPEC/SWARM.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Swarm API
 
 * [swarm.addrs](#swarmaddrs)
+* [swarm.localAddrs](#swarmlocaladdrs)
```
Comment (alanshaw) [inline SPEC/SWARM.md]: Please can this be arranged alphabetically? i.e. `swarm.localAddrs` should be after `swarm.disconnect` and before `swarm.peers`.

File: SPEC/SWARM.md [hunk 2]
```diff
@@ -30,6 +31,29 @@ ipfs.swarm.addrs(function (err, peerInfos) {
 
 A great source of [examples][] can be found in the tests for this API.
 
+#### `swarm.localAddrs`
+
+> List of known local addresses of each peer connected.
+
+##### `ipfs.swarm.localAddrs([callback])`
+
+`callback` must follow `function (err, peerInfos) {}` signature, where `err` is an error if the operation was not successful. `peerInfos` will be an array of [`PeerInfo`](https://github.com/libp2p/js-peer-info)s.
```
Comment (alanshaw) [inline SPEC/SWARM.md]: It actually returns [`Multiaddr`](https://github.com/multiformats/js-multiaddr/) instances _not_ `PeerInfo` instances.

File: SPEC/SWARM.md [hunk 3]
```diff
@@ -30,6 +31,29 @@ ipfs.swarm.addrs(function (err, peerInfos) {
 
 A great source of [examples][] can be found in the tests for this API.
 
+#### `swarm.localAddrs`
+
+> List of known local addresses of each peer connected.
```
Comment (alanshaw) [inline SPEC/SWARM.md]: Not quite correct, these are the addresses your node is listening on.

```suggestion
> List local addresses this node is listening on.
```

File: SPEC/SWARM.md (hunk 1 above)
Comment (Prabhakar-Poudel) [reply SPEC/SWARM.md]: Items in this list are already ordered logically than alphabetically. Should I fix them up too?

File: SPEC/SWARM.md (hunk 3 above)
Comment (Prabhakar-Poudel) [reply SPEC/SWARM.md]: Fixed.

File: SPEC/SWARM.md (hunk 2 above)
Comment (Prabhakar-Poudel) [reply SPEC/SWARM.md]: Fixed.

File: SPEC/SWARM.md (hunk 1 above)
Comment (alanshaw) [reply SPEC/SWARM.md]: Yes please

**Pull request merged**